integration-tests/artifacts/installer/v1alpha1/intaller.go

```diff
@@ -0,0 +1,174 @@
+package v1alpha1
```

Comment:
Please check the name of this .go file. Spellcheck!

Reply:
done